test/distributed_remote_server_control_commands_test.rb

```diff
@@ -27,7 +27,7 @@ def test_info
 
   def test_info_commandstats
     target_version "2.5.7" do
-      r.nodes.each { |n| n.config(:resetstat) }
+      r.nodes.each { |n| n.call(%i[config resetstat]) }
```

Comment:
This is a compatibility-breaking change. The return value's instance type is changed `Redis` into `Redis::Client`.